frontend/packages/ux-editor/src/components/FormField/FormField.module.css

```diff
@@ -1,3 +1,18 @@
+.container {
+  display: flex;
+  gap: 8px;
+  align-items: flex-end;
```

Comment:
Yep, I agree. I tried to do something clever here, but since most of the components from the design system have labels associated with them within the component, it was hard to do this in a good way. The <Label> component does not seem to have a `for`-prop, and `htmlFor` did not solve the problem.

Deferring this issue to future-team-studio to look at 😇